prediction_market_agent/agents/think_thoroughly_agent/deploy.py

```diff
@@ -1,84 +1,19 @@
-import typing as t
-from datetime import timedelta
-
-from loguru import logger
-from prediction_market_agent_tooling.config import PrivateCredentials
 from prediction_market_agent_tooling.deploy.agent import Answer, DeployableAgent
 from prediction_market_agent_tooling.markets.agent_market import AgentMarket
-from prediction_market_agent_tooling.markets.manifold.api import (
-    get_authenticated_user,
-    get_manifold_bets,
-    get_manifold_market,
-)
-from prediction_market_agent_tooling.markets.manifold.manifold import (
-    ManifoldAgentMarket,
-)
 from prediction_market_agent_tooling.markets.markets import MarketType
-from prediction_market_agent_tooling.markets.omen.omen import OmenAgentMarket
-from prediction_market_agent_tooling.markets.omen.omen_subgraph_handler import (
-    OmenSubgraphHandler,
-)
-from prediction_market_agent_tooling.tools.is_predictable import is_predictable_binary
-from prediction_market_agent_tooling.tools.utils import should_not_happen, utcnow
 
 from prediction_market_agent.agents.think_thoroughly_agent.think_thoroughly_agent import (
     CrewAIAgentSubquestions,
 )
-from prediction_market_agent.utils import APIKeys
 
 
 class DeployableThinkThoroughlyAgent(DeployableAgent):
     model: str = "gpt-4-turbo-2024-04-09"
-    max_markets_per_run = 5
+    bet_on_n_markets_per_run = 1
```

Comment:
to make it cheaper?

Reply:
yeah, so far, he hasn't proved himself to be so good to be so expensive 😄 or do you think we should just keep it?

Reply:
No I think that's okay - our openai bill is getting out of hand a bit, so I'll do the same for the mech agents